backend/custom_models/migrations/0007_upload_emib_sample_test.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.1.7 on 2019-06-03 15:27
+
+from django.db import migrations
+
+
+def upload_emib_sample(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_text = apps.get_model("custom_models", "ItemText")
+    test = apps.get_model("custom_models", "Test")
+    # get db alias
+    db_alias = schema_editor.connection.alias
+
+    # get emib_sample item
+    l_english = language.objects.using(db_alias).filter(
```

Comment:
how do you connect the french with this test as well?

Reply:
I'm actually connecting the item. But I'm finding the item from the english item_text. I could just as easily use the french. I use "emib_sample_item_text_en.**item_id**" which is the item not "emib_sample_item_text_en" which is the item_text.

I don't want to rely on the emib sample being the **only** item of type test, or even the first item of type test, thus why I look up the item_text.

The en/fr item_texts are already connected to the item.